README.md

```diff
@@ -219,6 +219,17 @@ CPMAddPackage(
 )
 ```
 
+Bazel is also supported, simply add to your WORKSPACE file:
+```
+git_repository(
+    name = "magic_enum",
+    commit = <>,
+    remote = "https://github.com/Neargye/magic_enum",
+)
+```
+To use bazel inside the repository it's possible to do:
+`export CC=clang ; bazel build //... ; bazel test //... ; bazel run //:example`
```

Comment:
Specifying `clang` is confusing, since right below it says that multiple compilers are supported.

Reply:
I updated the comment, please let me know what you think.
I initially ran into an issue because the system-default compiler is not supported, but you're right that any supported can be used.